src/App.js

```diff
@@ -1,15 +1,26 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Audio from './components/Audio';
 import IntroPageWrapper from './components/IntroPage';
-
+import YearBookIcon from './components/YearBookIcon';
 function App() {
+  //toggler for audio Icon
+    const [toggle, setToggle]= useState(false);
+  const toggler =() => {
+    toggle ? setToggle(false):setToggle(true);
+  };
+  //yearbook icon
+  function handleClick(e) {
+    e.preventDefault();
+  }
   return (
-    <div className="App">
+    <div>
+      <YearBookIcon onClick={handleClick}/>
```

Comment:
Similar to previous comment-- onClick is already being handled within the component